frame/tips/README.md

```diff
@@ -0,0 +1,118 @@
+# Treasury Module
+
+The Treasury module provides a "pot" of funds that can be managed by stakeholders in the
+system and a structure for making spending proposals from this pot.
+
+- [`treasury::Trait`](https://docs.rs/pallet-treasury/latest/pallet_treasury/trait.Trait.html)
+- [`Call`](https://docs.rs/pallet-treasury/latest/pallet_treasury/enum.Call.html)
+
+## Overview
+
+The Treasury Module itself provides the pot to store funds, and a means for stakeholders to
+propose, approve, and deny expenditures. The chain will need to provide a method (e.g.
+inflation, fees) for collecting funds.
+
+By way of example, the Council could vote to fund the Treasury with a portion of the block
+reward and use the funds to pay developers.
+
+### Tipping
+
+A separate subsystem exists to allow for an agile "tipping" process, whereby a reward may be
+given without first having a pre-determined stakeholder group come to consensus on how much
+should be paid.
+
+A group of `Tippers` is determined through the config `Trait`. After half of these have declared
+some amount that they believe a particular reported reason deserves, then a countdown period is
+entered where any remaining members can declare their tip amounts also. After the close of the
+countdown period, the median of all declared tips is paid to the reported beneficiary, along
+with any finders fee, in case of a public (and bonded) original report.
+
+### Bounty
```

Comment:
clean up docs here too

Reply:
Thanks for the comment, its my bad, overlooked readme.md